### ISSUE TITLE
user manual of ascii doctor is not available in PDF form on the website

### ISSUE THREAD

**Issue by zaxebo1** [asciidoctor/asciidoctor.org#407]
asciidoc-syntax-quick-reference.pdf and asciidoc-writers-guide.pdf are available in the PDF form , But user-manual is NOT available in PDF format

I tried to build it myself using asciidoctor-pdf , but it failed to build the pdf

Note that on phablet and tablets, it is easier to store and carry manuals in PDF format, and read them at leisure; compared to locally storing and reading html files on tablets.

So, Please PDF version of user manual available on the mainwebsite for downloading.

(this issue has been migrated from https://github.com/asciidoctor/asciidoctor/issues/1391 to here )

**Comment by zaxebo1** [comment]
the command 
```
asciidoctor-pdf --trace user-manual.adoc
```

gives error 
 i have pasted the error log at  http://pastebin.com/StCBHmWT

**Comment by zaxebo1** [comment]
The error log pasted at http://pastebin.com/StCBHmWT for better viewing , is now being pasted here also for ready reference

```
	

    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/zen-screenshot.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/sect0-title.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/sect0-title.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/doctitle.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/doctitle.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/author-email.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/author-email.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/author-compound.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/author-compound.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/author-email-long.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/author-email-long.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/multi-author.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/multi-author.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/rev.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/rev.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/rev-long.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/rev-long.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/preamble.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/preamble.png
    Failed to parse formatted text: When the <code>sectanchors</code> attribute is enabled on a document, an anchor (empty link) is added before the section title. The default Asciidoctor stylesheet renders the anchor as a section entity (<code>&sect;</code>) that floats to the left of the section title.
    Failed to parse formatted text: decimal&ast;
    Failed to parse formatted text: lowergreek&ast;
    Failed to parse formatted text: decimal&ast;
    Failed to parse formatted text: lowergreek&ast;
    Failed to parse formatted text: &ast; These styles are only supported by the HTML converters.
    Failed to parse formatted text: [arabic]&ast;
    Failed to parse formatted text: [arabic]&ast;
    Failed to parse formatted text: &ast; Default numeration if block style is not specified
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/url.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/sunset.jpg
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/sunset.jpg
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/sunset.jpg
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/sunset.jpg
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/icons/play.png
    asciidoctor: WARNING: image to embed not found or not readable: /home/user1/Desktop/images/icons/pause.png
    asciidoctor: WARNING: image to embed not found or not readable: http://upload.wikimedia.org/wikipedia/commons/3/35/Tux.svg
    asciidoctor: WARNING: allow-uri-read is not enabled; cannot embed remote image: http://upload.wikimedia.org/wikipedia/commons/3/35/Tux.svg
    asciidoctor: WARNING: image to embed not found or not readable: http://upload.wikimedia.org/wikipedia/commons/3/35/Tux.svg
    asciidoctor: WARNING: allow-uri-read is not enabled; cannot embed remote image: http://upload.wikimedia.org/wikipedia/commons/3/35/Tux.svg
    asciidoctor: WARNING: image to embed not found or not readable: http://upload.wikimedia.org/wikipedia/commons/3/35/Tux.svg
    Failed to parse formatted text: For example, to render the <code>&sect;</code> symbol you could write <code>&amp;sect;</code>, <code>&amp;#x00A7;</code>, or <code>&amp;#167;</code>. When the document is processed, <code>replacements</code> will replace the section symbol reference, regardless of whether it is a character entity reference or a numeric character reference, with <code>&amp;#167;</code>. In turn, <code>&amp;#167;</code> will render as <code>&sect;</code>.
    Failed to parse formatted text: <code>&amp;dagger;</code> renders as &dagger;
    Failed to parse formatted text: <code>&amp;euro;</code> renders as &euro;
    Failed to parse formatted text: <code>&amp;loz;</code> renders as &loz;
    Failed to parse formatted text: <code>&amp;#x25CA;</code> or <code>&amp;#9674;</code> render as &#x25CA;
    Failed to parse formatted text: <code>&amp;dagger;</code> renders as &dagger;
    Failed to parse formatted text: <code>&amp;euro;</code> renders as &euro;
    Failed to parse formatted text: <code>&amp;loz;</code> renders as &loz;
    Failed to parse formatted text: <code>&amp;#x25CA;</code> or <code>&amp;#9674;</code> render as &#x25CA;
    Failed to parse formatted text: The markup <u>underline <strong>me</strong></u> renders as underlined text and “me” is bold.
    Failed to parse formatted text: The markup <u>underline <strong>me</strong></u> renders as underlined text and “me” is bold.
    /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:2129:in `preserve_indentation': undefined method `gsub' for nil:NilClass (NoMethodError)
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:876:in `convert_listing_or_literal'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:88:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `block in content'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `map'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `content'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:104:in `convert_content_for_block'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:305:in `convert_section'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:88:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `block in content'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `map'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `content'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:104:in `convert_content_for_block'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:305:in `convert_section'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:88:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `block in content'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `map'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `content'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:104:in `convert_content_for_block'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:305:in `convert_section'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:88:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `block in content'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `map'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:80:in `content'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/document.rb:1086:in `content'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:104:in `convert_content_for_block'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:149:in `convert_document'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/lib/asciidoctor-pdf/converter.rb:88:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/document.rb:1028:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor.rb:1499:in `convert'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/cli/invoker.rb:94:in `block in invoke!'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/cli/invoker.rb:86:in `each'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-1.5.2/lib/asciidoctor/cli/invoker.rb:86:in `invoke!'
            from /home/user1/opt2_l/ruby/lib/ruby/gems/2.2.0/gems/asciidoctor-pdf-1.5.0.alpha.8/bin/asciidoctor-pdf:27:in `<top (required)>'
            from /home/user1/opt2_l/ruby/bin/asciidoctor-pdf:23:in `load'
            from /home/user1/opt2_l/ruby/bin/asciidoctor-pdf:23:in `<main>'

```

**Comment by zaxebo1** [comment]
The images warning part in the above error log can be ignored, but there are many "Failed to parse" messages in error log in midway. Probably they are the problem.

**Comment by mojavelinux** [comment]
I've fixed most of these errors in master. Asciidoctor PDF 1.5.0.alpha.9 will be released this weekend to address them. Then, we'll give it another shot.

**Comment by zaxebo1** [comment]
Just for the sake of completeness of the test, today i tried to convert usermanual to PDF by using **asciidoctorj** . 

it gave these errors ,which is pasted at http://pastebin.com/awk48UFP

"asciidoctorj --version"  shows 
version 1.5.2. 
It does not what is the version of asciidoctorj-pdf bundled with it

**Comment by jaredmorgs** [comment]
What do we need to do to move this issue forward for @zaxebo1 , @mojavelinux ?

**Comment by mojavelinux** [comment]
Asciidoctor PDF is not yet able to generate the user manual, so we're waiting on fixes upstream. I don't want to use fopub because that is a step backwards. So, for now, we have to wait.

**Comment by mojavelinux** [comment]
Duplicate of #328.

**Comment by Emonshr** [comment]
Any update?

**Comment by Emonshr** [comment]
Thanks for your kind information !

**Comment by mojavelinux** [comment]
Now that we've completed the transition to Antora, it is now possible to generate a PDF using https://gitlab.com/opendevise/oss/antora-site-generator-with-pdf-exporter. That software is still a prototype and may one day be included as part of Antora, but this can be a great test case for it.

**Comment by mojavelinux** [comment]
I'm going to transition this issue to the repository where that must be configured.
